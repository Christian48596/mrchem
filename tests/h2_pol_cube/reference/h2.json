{
  "input": {
    "constants": {
      "angstrom2bohrs": 1.8897261246257702,
      "dipmom_au2debye": 2.5417464739297717,
      "electron_g_factor": -2.00231930436256,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            -0.7001435291738478
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            0.7001435291738478
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 8,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "h2",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {
      "ext_el-0.000000": {
        "components": [
          {
            "initial_guess": {
              "file_CUBE_x_a": "cube_vectors/CUBE_x_a_0_vector.json",
              "file_CUBE_x_b": "cube_vectors/CUBE_x_b_0_vector.json",
              "file_CUBE_x_p": "cube_vectors/CUBE_x_p_0_vector.json",
              "file_CUBE_y_a": "cube_vectors/CUBE_y_a_0_vector.json",
              "file_CUBE_y_b": "cube_vectors/CUBE_y_b_0_vector.json",
              "file_CUBE_y_p": "cube_vectors/CUBE_y_p_0_vector.json",
              "file_chk_x": "checkpoint/X_rsp_0",
              "file_chk_y": "checkpoint/Y_rsp_0",
              "file_x_a": "initial_guess/X_a_rsp_0",
              "file_x_b": "initial_guess/X_b_rsp_0",
              "file_x_p": "initial_guess/X_p_rsp_0",
              "file_y_a": "initial_guess/Y_a_rsp_0",
              "file_y_b": "initial_guess/Y_b_rsp_0",
              "file_y_p": "initial_guess/Y_p_rsp_0",
              "prec": 0.001,
              "type": "cube"
            }
          },
          {
            "initial_guess": {
              "file_CUBE_x_a": "cube_vectors/CUBE_x_a_1_vector.json",
              "file_CUBE_x_b": "cube_vectors/CUBE_x_b_1_vector.json",
              "file_CUBE_x_p": "cube_vectors/CUBE_x_p_1_vector.json",
              "file_CUBE_y_a": "cube_vectors/CUBE_y_a_1_vector.json",
              "file_CUBE_y_b": "cube_vectors/CUBE_y_b_1_vector.json",
              "file_CUBE_y_p": "cube_vectors/CUBE_y_p_1_vector.json",
              "file_chk_x": "checkpoint/X_rsp_1",
              "file_chk_y": "checkpoint/Y_rsp_1",
              "file_x_a": "initial_guess/X_a_rsp_1",
              "file_x_b": "initial_guess/X_b_rsp_1",
              "file_x_p": "initial_guess/X_p_rsp_1",
              "file_y_a": "initial_guess/Y_a_rsp_1",
              "file_y_b": "initial_guess/Y_b_rsp_1",
              "file_y_p": "initial_guess/Y_p_rsp_1",
              "prec": 0.001,
              "type": "cube"
            }
          },
          {
            "initial_guess": {
              "file_CUBE_x_a": "cube_vectors/CUBE_x_a_2_vector.json",
              "file_CUBE_x_b": "cube_vectors/CUBE_x_b_2_vector.json",
              "file_CUBE_x_p": "cube_vectors/CUBE_x_p_2_vector.json",
              "file_CUBE_y_a": "cube_vectors/CUBE_y_a_2_vector.json",
              "file_CUBE_y_b": "cube_vectors/CUBE_y_b_2_vector.json",
              "file_CUBE_y_p": "cube_vectors/CUBE_y_p_2_vector.json",
              "file_chk_x": "checkpoint/X_rsp_2",
              "file_chk_y": "checkpoint/Y_rsp_2",
              "file_x_a": "initial_guess/X_a_rsp_2",
              "file_x_b": "initial_guess/X_b_rsp_2",
              "file_x_p": "initial_guess/X_p_rsp_2",
              "file_y_a": "initial_guess/Y_a_rsp_2",
              "file_y_b": "initial_guess/Y_b_rsp_2",
              "file_y_p": "initial_guess/Y_p_rsp_2",
              "prec": 0.001,
              "type": "cube"
            },
            "rsp_solver": {
              "checkpoint": false,
              "file_chk_x": "checkpoint/X_rsp_2",
              "file_chk_y": "checkpoint/Y_rsp_2",
              "final_prec": 1e-05,
              "helmholtz_prec": -1.0,
              "kain": 3,
              "max_iter": 10,
              "method": "DFT",
              "orbital_thrs": 0.01,
              "orth_prec": 1e-14,
              "property_thrs": -1.0,
              "start_prec": 1e-05
            }
          }
        ],
        "dynamic": false,
        "fock_operator": {
          "coulomb_operator": {
            "poisson_prec": 1e-05,
            "shared_memory": false
          },
          "exchange_operator": {
            "exchange_prec": -1.0,
            "poisson_prec": 1e-05
          },
          "xc_operator": {
            "shared_memory": false,
            "xc_functional": {
              "cutoff": 0.0,
              "functionals": [],
              "spin": false
            }
          }
        },
        "frequency": 0.0,
        "perturbation": {
          "operator": "h_e_dip",
          "r_O": [
            0.0,
            0.0,
            0.0
          ]
        },
        "properties": {
          "polarizability": {
            "pol-0.000000": {
              "frequency": 0.0,
              "operator": "h_e_dip",
              "precision": 1e-05,
              "r_O": [
                0.0,
                0.0,
                0.0
              ]
            }
          }
        },
        "unperturbed": {
          "fock_operator": {
            "coulomb_operator": {
              "poisson_prec": 1e-05,
              "shared_memory": false
            },
            "exchange_operator": {
              "exchange_prec": -1.0,
              "poisson_prec": 1e-05
            },
            "kinetic_operator": {
              "derivative": "abgv_55"
            },
            "nuclear_operator": {
              "proj_prec": 1e-05,
              "shared_memory": false,
              "smooth_prec": 1e-05
            },
            "xc_operator": {
              "shared_memory": false,
              "xc_functional": {
                "cutoff": 0.0,
                "functionals": [],
                "spin": false
              }
            }
          },
          "localize": false,
          "precision": 1e-05
        }
      }
    },
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-05,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-05
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-05,
          "shared_memory": false,
          "smooth_prec": 1e-05
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "screen": 12.0,
        "type": "sad",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-05,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        0.0
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.0017500315191592594,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -1.3894319402452746e-11,
            -1.3882874136323893e-11,
            0.0017500315191592594
          ],
          "vector_el": [
            0.0,
            0.0,
            0.001750031518898024
          ],
          "vector_nuc": [
            -1.3893166607865947e-11,
            -1.388201862558444e-11,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            -0.7001435291738478
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            0.7001435291738478
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -0.02418769188564947
        ],
        "occupation": [
          2.0
        ],
        "spin": [
          "p"
        ],
        "sum_occupied": -0.04837538377129894
      },
      "polarizability": {
        "pol-0.000000": {
          "frequency": 0.0,
          "isotropic_average": null,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "tensor": [
            null,
            null,
            null,
            null,
            null,
            null,
            -1.4531734534287122e-09,
            -1.453174603581884e-09,
            11.267735427097325
          ]
        }
      },
      "scf_energy": {
        "E_ee": 1.176968443209741,
        "E_eext": 0.0,
        "E_el": -1.225343824452135,
        "E_en": -3.289758121484827,
        "E_kin": 0.8874458538229509,
        "E_next": 0.0,
        "E_nn": 0.7141392859689609,
        "E_nuc": 0.7141392859689609,
        "E_tot": -0.5112045384831742,
        "E_x": 0.0,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 8,
      "routine": "mrchem.x",
      "total_cores": 8,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": {
      "ext_el-0.000000": {
        "components": [
          null,
          null,
          {
            "rsp_solver": {
              "converged": true,
              "cycles": [
                {
                  "mo_residual": 4.876875709450821,
                  "property_update": -8.447459028983616,
                  "symmetric_property": -8.447459028983616,
                  "wall_time": 6.4977425
                },
                {
                  "mo_residual": 1.1086096638666987,
                  "property_update": -3.643203039011043,
                  "symmetric_property": -12.090662067994659,
                  "wall_time": 4.528499375
                },
                {
                  "mo_residual": 0.32379159420393866,
                  "property_update": 0.8256331017967806,
                  "symmetric_property": -11.265028966197878,
                  "wall_time": 4.923529833
                },
                {
                  "mo_residual": 0.002447885391557853,
                  "property_update": -0.0027064608994464834,
                  "symmetric_property": -11.267735427097325,
                  "wall_time": 4.91991875
                }
              ],
              "wall_time": 20.869818125
            }
          }
        ],
        "frequency": 0.0,
        "perturbation": "h_e_dip",
        "success": true
      }
    },
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 1.176968443209741,
        "E_eext": 0.0,
        "E_el": -1.225343824452135,
        "E_en": -3.289758121484827,
        "E_kin": 0.8874458538229509,
        "E_next": 0.0,
        "E_nn": 0.7141392859689609,
        "E_nuc": 0.7141392859689609,
        "E_tot": -0.5112045384831742,
        "E_x": 0.0,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
