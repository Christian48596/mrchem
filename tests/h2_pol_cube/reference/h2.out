                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         cube_guess_rsp                           ***
***         Git commit hash    60e30ef5e7e906bd2411-dirty               ***
***         Git commit author  ggerez                                   ***
***         Git commit date    Wed Oct 26 21:46:01 2022 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (8 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 8
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000       -0.700144
    1       H            :        0.000000        0.000000        0.700144
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999999738616e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  0.887445853823
 E-N energy              :            (au)                 -3.289758121485
 Coulomb energy          :            (au)                  1.176968443210
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                  0.000000000000
 N-N energy              :            (au)                  0.714139285969
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.225343824452
 Nuclear energy          :            (au)                  0.714139285969
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.112045384832e-01
                         :      (kcal/mol)             -3.207856910822e+02
                         :        (kJ/mol)             -1.342167331488e+03
                         :            (eV)             -1.391058414737e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.024187691886
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -0.048375383771
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***               Computing Linear Response Wavefunction                ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
Info: /Users/gsa017/repos/mrchem/src/driver.cpp: guess_orbitals(), line 862: guess type cube
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project cube file molecular orbitals
 Precision               : 1.00000e-03
 Restricted              : True
 MO file                 : cube_vectors/CUBE_x_p_0_vector.json
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Info: /Users/gsa017/repos/mrchem/src/driver.cpp: guess_orbitals(), line 864: 1
===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              4.893547500306e+00
===========================================================================


Info: /Users/gsa017/repos/mrchem/src/driver.cpp: run(), line 767: false
Info: /Users/gsa017/repos/mrchem/src/driver.cpp: guess_orbitals(), line 862: guess type cube
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project cube file molecular orbitals
 Precision               : 1.00000e-03
 Restricted              : True
 MO file                 : cube_vectors/CUBE_x_p_1_vector.json
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Info: /Users/gsa017/repos/mrchem/src/driver.cpp: guess_orbitals(), line 864: 1
===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              4.893547500306e+00
===========================================================================


Info: /Users/gsa017/repos/mrchem/src/driver.cpp: run(), line 767: false
Info: /Users/gsa017/repos/mrchem/src/driver.cpp: guess_orbitals(), line 862: guess type cube
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project cube file molecular orbitals
 Precision               : 1.00000e-03
 Restricted              : True
 MO file                 : cube_vectors/CUBE_x_p_2_vector.json
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Info: /Users/gsa017/repos/mrchem/src/driver.cpp: guess_orbitals(), line 864: 1
===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              5.979061213753e+00
===========================================================================


Info: /Users/gsa017/repos/mrchem/src/driver.cpp: run(), line 767: false
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize linear response orbitals
 Frequency               : Static field
 Perturbation            : h_E_dip[z]
 Method                  : DFT
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 10
 KAIN solver             : 3
 Start precision         : 1.00000e-05
 Final precision         : 1.00000e-05
 Helmholtz precision     : Dynamic
 Property threshold      : Off
 Orbital threshold       : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Info: /Users/gsa017/repos/mrchem/src/scf_solver/LinearResponseSolver.cpp: optimize(), line 73: (-1.717327e+00,0.000000e+00)
===========================================================================
 Iter           MO residual       Symmetric property                Update
---------------------------------------------------------------------------
    0          1.000000e+00           0.000000000000          0.000000e+00
    1          4.876876e+00          -8.447459028984         -8.447459e+00
    2          1.108610e+00         -12.090662067995         -3.643203e+00
    3          3.237916e-01         -11.265028966198          8.256331e-01
    4          2.447885e-03         -11.267735427097         -2.706461e-03
---------------------------------------------------------------------------
                      SCF converged in 4 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000       -0.700144
    1       H            :        0.000000        0.000000        0.700144
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  0.887445853823
 E-N energy              :            (au)                 -3.289758121485
 Coulomb energy          :            (au)                  1.176968443210
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                  0.000000000000
 N-N energy              :            (au)                  0.714139285969
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.225343824452
 Nuclear energy          :            (au)                  0.714139285969
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.112045384832e-01
                         :      (kcal/mol)             -3.207856910822e+02
                         :        (kJ/mol)             -1.342167331488e+03
                         :            (eV)             -1.391058414737e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.024187691886
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -0.048375383771
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000        0.001750
 Magnitude               :            (au)                        0.001750
                         :         (Debye)                        0.004448
---------------------------------------------------------------------------
 Nuclear vector          :       -0.000000       -0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :       -0.000000       -0.000000        0.001750
 Magnitude               :            (au)                        0.001750
                         :         (Debye)                        0.004448
===========================================================================


===========================================================================
                       Polarizability (pol-0.000000)
---------------------------------------------------------------------------
 Frequency               :            (au)                        0.000000
                         :          (cm-1)                        0.000000
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Total tensor            :             N/A             N/A             N/A
                         :             N/A             N/A             N/A
                         :       -0.000000       -0.000000       11.267735
 Isotropic average       :            (au)                             N/A
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 30s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
